_data/accessibility-tests/footer.yml

```diff
@@ -0,0 +1,97 @@
+title: Footer
+component_status: pass
+test_items:
+# General tests
+  - summary: Some fields have auto-populate capability. 
+    summary_additional: When you navigate to an input field in the footer (like an email address), there will be an option in the browser to auto-populate information.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.5
```

Comment:
This criterion is still in discussion. Internal [comment thread](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?disco=AAABPzefgUw) (🔒)

Reply:
I sent some suggested changes for this check in our spreadsheet. I think this criteria partially conforms or could conform with exceptions as there are two conditions to pass this check. Ours passes the first but not the second condition:

The input field serves a purpose identified in the [Input Purposes for user interface components section](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html#input-purposes); (PASS)

and
The content is implemented using technologies with support for identifying the expected meaning for form input data. (FAIL)

The check can also be conditional since not all data input fields in the footer will need to collect USER data. Some may be more generic so we can't know what kind of data field may appear in the footer.

Reply:
@amycole501 @alex-hull @mahoneycm 

**suggestion (blocking):** This item feels difficult to test because it appears appears to be a browser-dependent feature. Even the [autocomplete docs in MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) do not seem to work in Firefox and Safari (Please correct me if I'm wrong there). I'm wondering if we can re-write this to match the other 1.3.5 tests _for now_ until we can get a clearer stance and provide better guidance around the `autocomplete` attribute. 

```suggestion
  - summary: In the big footer variant, the email input field announces its purpose.
    summary_additional: When you navigate to the email address input field in the big footer variant, you hear that the input expects an email address.
    test_status: pass
    test_type: screen_reader
    version_tested: 3.8.1
    wcag_criterion: 1.3.5
```

Reply:
I think it makes sense to draft it after 1.3.5. I think it is different enough from criteria like info and relationships that it can be worded this way.